shinydashboard/lantern/server.R

```diff
@@ -16,4 +20,56 @@ function(input, output, session) {
    callModule(
     performance,
     "performance_page") 
-}
\ No newline at end of file
+   
+   callModule(
+     capabilitymodule,
+     "capability_page",
+     reactive(input$fhir_version),
+     reactive(input$vendor))
+   
+   page_name_list <- list("dashboard_tab" = "Current Endpoint Metrics",
+                          "endpoints_tab" = "List of Endpoints",
+                          "capability_tab" = "Capability Page",
+                          "availability_tab" = "Endpoint Server Availability",
+                          "performance_tab" = "Mean Response Time",
+                          "location_tab" = "Location Map Page",
+                          "about_tab" = "About Lantern"
+                        )
+   show_filter <- reactive(input$side_menu %in% c("endpoints_tab","capability_tab"))
+   
+  page_name <- reactive({
+    page_name_list[[input$side_menu]]
+  })
+   
+  output$page_title <- renderText(page_name())
+   
+   observeEvent(input$main_tabs,{print(input$main_tabs)
+     print(show_filter())
+   })
+   
+   output$show_filters <- renderUI({
+     if (show_filter()) {
+       fluidRow(
+         column(width = 4,
+                      selectInput(
+                        inputId = "fhir_version",
+                        label = "FHIR Version:",
+                        choices = fhir_version_list,
+                        # selected = ui_special_values$ALL_FHIR_VERSIONS,
+                        size = 1,
+                        selectize = FALSE)
+         ),
+         column(width=4,
+                      selectInput(
+                        inputId = "vendor",
+                        label = "Vendor:",
+                        choices = vendor_list,
+                       #  selected = ui_special_values$ALL_VENDORS,
```

Comment:
i'm assuming these two commented out lines are no longer being used? but if i'm missing something because i don't know R very well and you want to keep them that's fine!

Reply:
uncommented lines